tests/unit_tests/test_drop_dimension.py

```diff
@@ -0,0 +1,166 @@
+import json
+
+import pytest
+import subprocess
+
+from tests.utils import load_process_code, load_datacube_code, run_process
+
+
+@pytest.fixture
+def drop_dimension_process_code():
+    return load_process_code("drop_dimension")
+
+
+@pytest.mark.parametrize(
+    "data,name,additional_js_code_specific_to_case,expected_output",
+    [
+        (
+            {"B01": [1, 2, 3]},
```

Comment:
Btw can we change all these `data` values as they are not really valid. I noticed this got introduced in some other PRs. Namely, band values will never be arrays, but numbers. Additionally, the case where we don't have temporal script also isn't properly implemented, so we can expect the input `samples` will be arrays of objects.

Like this:
`{"B01": [1, 2, 3]}` => `[{"B01": 1}]`

-------------------------------------

Btw, the last argument in `DataCube` constructor is a bool `fromSamples`, which says if the input data should be converted from the above format to arrays of arrays or should it set it directly. If `false`, you can set it directly.
e.g.
```js
new Datacube([[[[1,2,3,4]], [[5,6,7,8]]], [[[9,10,11,12]], [[13,14,15,16]]]], 't', 'b', false)
```
However, I don't recommend doing that in these tests as we soon won't be using arrays of arrays anymore. Just generally so you're aware.

Reply:
Ok I will fix these incorrect `data` inputs.

Thanks for the info!

Reply:
Ah I see, sorry I misunderstood. Reverted that commit so the changes are only for this file.